src/types/posters.ts

```diff
@@ -1,6 +1,15 @@
 export type Poster = {
-  title: string;
-  categories: string[];
-  introVideoUrl: string;
   authorName: string;
+  categories: string[];
+  title: string;
+  authors?: string[];
+  contactEmail?: string;
+  introVideoUrl?: string;
+  thumbnailUrl?: string;
+  keywords?: string[];
+  introduction?: string;
+  moreInfoUrl?: string;
+  moreInfoUrls?: string[];
+  moreInfoUrlTitle?: string;
+  posterId?: string;
 };
```

Comment:
The fields added here seem kind of suboptimal and redundant.. definitely needs to be cleaned up in future.